docs/F-Refinements/pointers-arrays-and-structs.md

```diff
@@ -164,9 +171,9 @@ The above program produces the following output:
 5611
 ```
 
-Caution
-
+:::caution
 The syntactic equivalence between pointer parameters and array names does not extend to the definition of an array.  We cannot replace the definition of an array with a pointer definition.  An array definition allocates the stated number of memory locations for all of the elements in the array.  A pointer definition **allocates only one memory location to hold a single address**. 
```

Comment:
There are a couple of places in all of the file where after a sentence, you can see a an extra space.
Please check the image below:

![Screen Shot 2021-11-19 at 10 48 12 PM](https://user-images.githubusercontent.com/69704498/142713484-66bd210f-2c49-4160-b070-eb915eb80ece.png)

It's an easy fix

Reply:
One thought for this! I think it happened because a writer who first wrote it put the sentences in a different row, and in markdown, it considers as another single space unless you put another line. It considers different lines without an extra next line in one single paragraph.

Reply:
fixed